libretroshare/src/retroshare/rsgxschannels.h

```diff
@@ -282,26 +283,6 @@ class RsGxsChannels: public RsGxsIfaceHelper, public RsGxsCommentService
 	 */
 	virtual bool ExtraFileRemove(const RsFileHash& hash) = 0;
 
-	/**
-	 * @brief Get auto-download option value for given channel
-	 * @jsonapi{development}
```

Comment:
they are, I have just deprecated them because they use the "service string" internally in a  terrible manner, and when you enable those features retroshare start acting weird, flooding the log with error messages etc, a few months ago I also investigated if it was possible to fix that functionality, but I discovered the terrible truth...

Reply:
well then it's better to put a comment about why it's deprecated. Because now it looks like there's a replacement but there isn't.